_includes/devdoc/guide_transactions.md

```diff
@@ -47,7 +47,7 @@ invalidating previous transactions.
 ![Spending An Output](/img/dev/en-tx-overview-spending.svg)
 
 An output has an implied index number based on its location in the
-transaction---the first output is output zero. The output also has an
+transaction---the index of first output is zero. The output also has an
```

Comment:
The word "the" is missing. Please change to: "the index of *the* first output"

Also, do not italicize; I only added it above to emphasis the change.